Vic2ToHoI4/Data_Files/blankMod/output/common/ideas/_converter.txt

```diff
@@ -0,0 +1,230 @@
+#Vanilla HoI4 ideas
+#Keep allowed always=no for all of them
+ideas = {
+
+	country = {
+		general_staff = {
+			
+			allowed = {
+				always = no
+			}
+
+			allowed_civil_war = {
+				always = yes
+			}
+
+			removal_cost = -1
+			
+			modifier = {
+				army_org_Factor = 0.05
+				planning_speed = 0.25
+			}
+		}	
+		CHI_ineffective_bureaucracy = {
+
+			picture = generic_disjointed_gov
+		
+			allowed = {
+				always = no
+			}
+
+			allowed_civil_war = {
+				always = yes
+			}
+
+			removal_cost = -1
+			
+			modifier = {
+				conscription_factor = -0.35
+			}
+
+		}
+		MAN_banditry = {
+
+			allowed = {
+				always = no
+			}
+
+			allowed_civil_war = {
+				original_tag = MAN

```

Comment:
We should avoid ideas that have hard-coded tags, as those tags may not be present in converted games.